webplugin/js/app/media/media-service.js

```diff
@@ -1,6 +1,37 @@
 Kommunicate.mediaService = {
     browserLocale:
-       window.navigator.language || window.navigator.userLanguage || 'en-us',
+        window.navigator.language || window.navigator.userLanguage || 'en-US',
+    isAppleDevice: function () {
+        var isIOSDevice = /iPhone|iPad|iPod/i.test(navigator.userAgent);
+        var isMacSafari = /^((?!chrome|android).)*safari/i.test(
+            navigator.userAgent
+        );
+
+        return isIOSDevice || isMacSafari;
+    },
+    endSttExplicitly: function (
+        lastListeningEventTime,
+        recognizingDone,
+        recognition,
+        checkLastSpeech
+    ) {
+        const MILLISECOND_TO_SEC = 1000;
+        const CUSTOM_TIMEOUT_STT = 5; //To do set this via widget script
```

Comment:
move it to app option settings

Reply:
Done, getting this from widget script also by default time is 10sec